public/home.html

```diff
@@ -49,8 +49,8 @@ <h1>Easy RPC</h1>
     <br>
     <input type="text" name="profileName" id="profileName" class="input-feild" placeholder="Profile Name">
     <input type="text" name="token" id="token" class="input-feild" placeholder="Client ID"><br>
-    <input type="text" name="state" id="state" class="input-feild" placeholder="State">
-    <input type="text" name="details" id="details" class="input-feild" placeholder="Details"><br>
+    <input type="text" name="details" id="details" class="input-feild" placeholder="Top text">
+    <input type="text" name="state" id="state" class="input-feild" placeholder="Bottom text"><br>
```

Comment:
Could we keep the placeholders to `state` and `details` itself. They sound more formal :)